### ISSUE TITLE
Add argument to suppress runtime updates

### ISSUE THREAD

**Issue by xavpaice** [replicatedhq/troubleshoot#695]
In #694, we add the ability to specify a `uri`in the support bundle definition provided to Troubleshoot, to download an updated spec.

In design #681, we discuss having the option to disable that, based on a CLI switch.

## In scope

* CLI usage of Troubleshoot via the `support-bundle`binary, providing Viper arguments

## Out of scope

* Any calls to Troubleshoot other than that in package `cli`, `runTroubleshoot()`

## Definition of done

- [x] the Troubleshoot CLI accepts a `--follow-uri` boolean switch which disables any action taken for the `uri`field and sets `followURI` to `false`.  Default (i.e. switch not provided) is to download from the uri.
- [x] changes have tests
- [x] the new CLI option is documented in code (via `--help`)

**Comment by edgarlanting** [comment]
> `--ignore-uri` sounds fine to me, it's more specific than `--no-updates`.

Thanks @xavpaice I was initially going for `noupdates` but `ignore-uri` is more clear 👍🏻

**Comment by xavpaice** [comment]
`--ignore-uri` sounds fine to me, it's more specific than `--no-updates`.

**Comment by edgarlanting** [comment]
Current steps for implementation of the `--ignoreuri` flag are;

When passed as `support-bundle --ignoreuri <spec.yaml>` the flag will make sure that `followURI` is set to `false` as defined in ./pkg/supportbundle.load.go will be ignored:

```
		// check if there is a uri field and if so,
		// use the upstream spec, otherwise fall back to
		// what's defined in the current spec
		if supportBundle.Spec.Uri != "" && followURI {
			logger.Printf("using upstream reference: %+v\n", supportBundle.Spec.Uri)
			upstreamSupportBundleContent, err := LoadSupportBundleSpec(supportBundle.Spec.Uri)
			if err != nil {
				logger.Printf("failed to load upstream supportbundle, falling back")
				return supportBundle, nil
			}
....
```
I am using @adamancini's branch at: https://github.com/replicatedhq/troubleshoot/pull/702/files for reference.

~I expect to have a draft PR ready this Thursday for review.~

**Comment by edgarlanting** [comment]
Current status of this task:
 - Add functionality to `./pkg/supportbundle/load.go` in order to set `followURI` to `false`
 - Add functionality to `./cmd/troubleshoot/cli/root.go / run.go` in order to make the flag functional and be able to use it.

Working on testing out various ways of implementing this. Draft PR is therefore not ready yet today

Will follow up with examples of what I'm working on..

**Comment by xavpaice** [comment]
if it's easier to code, you could make the option `--follow-uri=false` and have Viper default that to true.

**Comment by edgarlanting** [comment]
Upon reviewing the current state of our CLI code (`run.go` & `root.go`) we found that using Viper to implement the new change was held back because we also redefine `v` in `run.go` at a point. 

Therefore we cannot reuse `v` for Viper references. This was resolved by refactoring the following on line `89` in `run.go`:

`for i, v := range arg {` -> `for i, val := range arg {`

And:

`collectorContent, err := supportbundle.LoadSupportBundleSpec(v)` -> `collectorContent, err := supportbundle.LoadSupportBundleSpec(val)`

_For learning references:_
https://golangdocs.com/for-loop-in-golang Describes the standard Go "for" loop definition.

Upon changing this, changing from using `ParseSupportBundleFromDoc` to use the function `ParseSupportBundle` on line `100` and adding a secondary argument (`follow-uri`), made sure the additional flag could be implemented.
